content/en/docs/making-requests.md

```diff
@@ -152,7 +153,7 @@ and the response is received. If an error occurs during the operation,
 it will be returned. The output or resulting structure won't be valid.
```

Comment:
"will be returned. The" -> "is returned and the output or resulting structure is not valid."

Reply:
Will fix this one in a follow-up PR if still relevant as I have rewritten most of `making-requests.md`